cmd/eksctl-anywhere/cmd/downloadimages.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+*/
+package cmd
+
+import (
+	"context"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-anywhere/cmd/eksctl-anywhere/cmd/internal/commands"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/docker"
+	"github.com/aws/eks-anywhere/pkg/executables"
+	"github.com/aws/eks-anywhere/pkg/version"
+)
+
+// imagesCmd represents the images command
+var downloadImagesCmd = &cobra.Command{
```

Comment:
If we're going to start a new cmd pattern (i.e. using `internal/commands`) can we begin using initializing constructors to create cmds in an effort to remove package level state that has promoted a noteworthy number of copy-n-paste mistakes.

```go
func NewDownloadImagesCmd() &cobra.Command { ... }
```

This approach much better compliments the ability to bind arguments and error out naturally too as there's no need for an `init` func that prevents anything other than an immediate exit.

Reply:
Ah sorry, was this a move of existing logic? I missed that if so! If we're just moving stuff around then its not worth changing.

With respect to Cobra cmds and 'compatibility' I wasn't suggesting we couldn't move this a second time, it was more should we begin a precedent in this PR.

Reply:
Spoke offline.

Will focus any attempts to adjust how we write commands in a dedicated refactor with design.